client/test/unitTests/Util/Auth/Authentication.test.ts

```diff
@@ -0,0 +1,28 @@
+import { signOut } from 'util/auth/Authentication';
+import { useAuth } from 'react-oidc-context';
+
+jest.mock('react-oidc-context');
```

Comment:
This https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/client/test/integration/authRedux.test.tsx#L10 test might be useful